snuba/web/views.py

```diff
@@ -379,32 +378,22 @@ def handle_subscription_error(exception: InvalidSubscriptionError):
     )
 
 
-def build_external_subscription_id(partition_id: str, subscription_id: str) -> str:
-    return "{}{}{}".format(partition_id, SUBSCRIPTION_SEPARATOR, subscription_id)
+def build_external_subscription_id(identifier: SubscriptionIdentifier) -> str:
+    return "{}{}{}".format(
+        identifier.partition_id, SUBSCRIPTION_SEPARATOR, identifier.subscription_id
+    )
```

Comment:
I would be OK with making this the `__str__` on `SubscriptionIdentifier` and just using that if you'd like.

Reply:
I was thinking about that... It felt weird for `SubscriptionIdentifier` to be aware of the format that the api expects the identifier to be in.

Reply:
I think that's reasonable too, I don't have a strong opinion about it and it's relatively easy to change. TBH, might just be worth inlining it for now as an f-fstring, unless you plan in reusing it — I think it'd be equally clear that way. Fine as-is too if you'd rather.